src/components/Nav/Mobile.js

```diff
@@ -244,15 +244,14 @@ const MobileNavMenu = ({
                   <SectionItems>
                     {section.items.map((item, idx) => {
```

Comment:
No need for the return statement. 
```
section.items.map((item, idx) => (
  <SectionItem key={idx} onClick={() => toggleMenu()}>
     ...
  </SectionItem>
)
```

Reply:
Agreed, these are scattered throughout the codebase and could be cleaned up in batch in a separate issue. I'll push the syntax fix here though while we're looking at it.